docs/ref/contrib/postgres/aggregates.txt

```diff
@@ -0,0 +1,235 @@
+=========================================
+PostgreSQL specific aggregation functions
+=========================================
+
+.. module:: django.contrib.postgres.aggregates
+   :synopsis: PosgreSQL specific aggregation functions
+
+All of these functions are available
+from the ``django.contrib.postgres.aggregates``.
+
+.. warning::
+
+    All functions come without default aliases,
+    so you must explicitly provide one.
+    Usage example::
+
+        >> result = SomeModel.objects.aggregate(arr=ArrayAgg('somefield'))
```

Comment:
>>> instead of >>
we can show the result directly as done in other examples instead of printing it